src/app/api/admin/SessionID/route.ts

```diff
@@ -0,0 +1,100 @@
+import { NextRequest ,NextResponse } from 'next/server';
+import AdminSessionID from "@/models/AdminSessionID";
+import clientPromise from "../../../../../lib/mongodb";
+import {v4 as uuidv4} from 'uuid';
+import { Db } from 'mongodb';
+
+export async function POST(req: NextRequest)
+{
+	try {
+		const res = await req.json();
+		const sessionID = uuidv4();
+		const adminOTP = new AdminSessionID({
+			sessionID,
+			adminEmail: res.adminEmail,
+			createdAt: new Date(),
+			expiresAt: new Date(Date.now() + 60 * 60 * 1000), 
+		});
+
+        const client = await clientPromise;
+        const db = client.db("42reef-check");
+		await db.collection("adminSessions").deleteMany({ adminEmail: res.adminEmail });
+        await db.collection("adminSessions").insertOne(adminOTP);
+
+		const response = NextResponse.json({ message: 'Logged in successfully' });
+        response.cookies.set('sessionID', sessionID, {
+            httpOnly: true,
+            secure: true,
+			sameSite: 'strict',
+            maxAge: 60 * 60, // 1 hour
+			expires: new Date(Date.now() + 60 * 60 * 1000), 
+            path: '/',
+        });
+
+		return response;
+    } catch (error) {
+        console.error('Error creating sessionID:', error);
+        return NextResponse.json({ message: "Failed to create or save sesionID to database."}, {status: 400});
+    }
+}
+
+export async function DELETE(req: NextRequest)
+{
+	try {
+		const sessionID = req.cookies.get('sessionID')?.value;
+
+		if (!sessionID) {
+			return NextResponse.json({ message: 'No session ID found' }, { status: 400 });
+		}
+
+        const client = await clientPromise;
+        const db = client.db("42reef-check");
+		const result = await db.collection("adminSessions").deleteMany({ sessionID });
+
+		if (result.deletedCount === 0) {
+			return NextResponse.json({ message: "SessionID not found" }, { status: 404 });
+		}
+
+		const response = NextResponse.json({ message: "SessionID deleted successfully" }, { status: 200 });
```

Comment:
@Zyeoh883  
for Line 58,  
i just realised it's actually two separate arguments (note the comma in between!)
![brave_ui3HiOAQTo](https://github.com/42KL-admin/42DoGood-Reef_Check/assets/20178082/de435caf-3020-4b6a-a785-53b36dc72f92)

hovering over the status property/key also shows it's part of `ResponseInit`, that's why we couldn't read `result.status` in the `DropdownMenu.tsx` file
![image](https://github.com/42KL-admin/42DoGood-Reef_Check/assets/20178082/edb86f80-bbf8-4fa9-a8a9-36daf4932d35)


`response.status` does work as expected
![image](https://github.com/42KL-admin/42DoGood-Reef_Check/assets/20178082/ee5ab540-a021-4e56-a5c2-06820db077eb)
![image](https://github.com/42KL-admin/42DoGood-Reef_Check/assets/20178082/0da9c2ae-3e57-4f71-a45f-abf1762eafe5)

this should solve the problem!